state/modelgeneration.go

```diff
@@ -218,15 +198,15 @@ func assignGenerationUnitTxnOps(id, appName, unitName string) []txn.Op {
 	}
 }
 
-// AutoComplete marks the generation as completed, if it is active and
-// meets autocomplete criteria.  Becomes the "current" generation.
+// AutoComplete marks the generation as completed if autocomplete criteria
+// is met.  It then becomes the "current" generation.
 func (g *Generation) AutoComplete() error {
 	err := g.complete(false)
```

Comment:
If AutoComplete already has the "only complete if we can", why make the CanAutocomplete check at all?

Reply:
`CanAutoComplete` was written before the `AutoComplete` logic got its form. I have now removed it as you suggested.